azext_iot/tests/central/test_iot_central_int.py

```diff
@@ -23,7 +23,7 @@
 APP_PRIMARY_KEY = os.environ.get("azext_iot_central_primarykey")
 APP_SCOPE_ID = os.environ.get("azext_iot_central_scope_id")
 DEVICE_ID = os.environ.get("azext_iot_central_device_id")
-TOKEN = os.environ.get("azext_iot_central_token")
+TOKEN = os.environ.get("azext_iot_central_token", None)
```

Comment:
`None` should be the default if the `azext_iot_central_token` key does not have a value.

Reply:
removed